src/pages/RFOX/components/Claim/ClaimSelect.tsx

```diff
@@ -0,0 +1,121 @@
+import { useHistory } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIconWithBadge } from 'components/AssetIconWithBadge'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText } from 'components/Text'
+import { TransactionTypeIcon } from 'components/TransactionHistory/TransactionTypeIcon'
+import { toBaseUnit } from 'lib/math'
+import { selectAssetById, selectFirstAccountIdByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { RfoxClaimQuote } from './types'
+import { ClaimRoutePaths, type ClaimRouteProps } from './types'
+
+type ClaimRowProps = {
+  assetId: AssetId
+  amount: string
+  status: string
+  setClaimQuote: (quote: RfoxClaimQuote) => void
+}
+
+const hoverProps = { bg: 'gray.700' }
+
+const ClaimRow: FC<ClaimRowProps> = ({ assetId, amount, status, setClaimQuote }) => {
+  const translate = useTranslate()
+  const history = useHistory()
+
+  const claimAsset = useAppSelector(state => selectAssetById(state, assetId))
+  const assetSymbol = claimAsset?.symbol
+  const claimAmountCryptoBaseUnit = toBaseUnit(bnOrZero(amount), claimAsset?.precision ?? 0)
+
+  // TODO(apotheosis): make this programmatic when we implement multi-account
+  const claimAssetAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, claimAsset?.chainId ?? ''),
+  )
+
+  const claimQuote: RfoxClaimQuote = useMemo(
+    () => ({
+      claimAssetAccountId: claimAssetAccountId ?? '',
+      claimAssetId: assetId,
+      claimAmountCryptoBaseUnit,
+    }),
+    [assetId, claimAmountCryptoBaseUnit, claimAssetAccountId],
+  )
+
+  const handleClick = useCallback(() => {
+    setClaimQuote(claimQuote)
+    history.push(ClaimRoutePaths.Confirm)
+  }, [claimQuote, history, setClaimQuote])
+
+  return (
+    <Flex
+      as={Button}
+      align='center'
+      variant='unstyled'
+      p={8}
+      borderRadius='md'
+      width='100%'
+      onClick={handleClick}
+      _hover={hoverProps}
+    >
+      <Box mr={4}>
+        <AssetIconWithBadge assetId={foxAssetId}>
+          <TransactionTypeIcon type={TransferType.Receive} />
+        </AssetIconWithBadge>
+      </Box>
+      <Box mr={4}>
+        <RawText fontSize='sm' color='gray.400' align={'start'}>
+          {translate('RFOX.unstakeFrom', { assetSymbol })}
+        </RawText>
+        <RawText fontSize='xl' fontWeight='bold' color='white' align={'start'}>
+          {assetSymbol}
+        </RawText>
+      </Box>
+      <Box flex='1' alignItems={'end'}>
+        <RawText fontSize='sm' fontWeight='bold' color='green.300' align={'end'}>
+          {status}
+        </RawText>
+        <RawText fontSize='xl' fontWeight='bold' color='white' align={'end'}>
+          <Amount.Crypto value={amount} symbol={assetSymbol ?? ''} />
+        </RawText>
+      </Box>
+    </Flex>
+  )
+}
+
+type ClaimSelectProps = {
+  setClaimQuote: (quote: RfoxClaimQuote) => void
```

Comment:
As well as here

Reply:
Actioned in https://github.com/shapeshift/web/pull/7004/commits/b39ea0eb699d2309beeb8cef365131bd0bba9bab